include/swift/Option/FrontendOptions.td

```diff
@@ -1257,6 +1257,11 @@ def enable_move_inout_stack_protector :
   Flag<["-"], "enable-move-inout-stack-protector">,
   HelpText<"Enable the stack protector by moving values to temporaries">;
 
+// This feature is enabled by default, the flag is a no-op.
+def enable_import_ptrauth_field_function_pointers :
+   Flag<["-"], "enable-import-ptrauth-field-function-pointers">,
+   HelpText<"Enable import of custom ptrauth qualified field function pointers">;
```

Comment:
Since this is defaulting on—do we want this hidden? (Or: I guess this is already in `--help-hidden`?)
Do we want the help message to mention it's default-on?

Reply:
I can update the help message. Thanks